freeipa/src/main/java/com/sequenceiq/freeipa/service/image/userdata/CcmV2TlsTypeDecider.java

```diff
@@ -1,22 +1,15 @@
 package com.sequenceiq.freeipa.service.image.userdata;
 
-import javax.inject.Inject;
-
 import org.springframework.stereotype.Component;
 
-import com.sequenceiq.cloudbreak.auth.altus.EntitlementService;
 import com.sequenceiq.common.api.type.CcmV2TlsType;
 import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
 
 @Component
 public class CcmV2TlsTypeDecider {
 
-    @Inject
-    private EntitlementService entitlementService;
-
     public CcmV2TlsType decide(DetailedEnvironmentResponse environment) {
-        return CcmV2TlsType.TWO_WAY_TLS == environment.getCcmV2TlsType() ||
-                environment.getCcmV2TlsType() == null && !entitlementService.ccmV2UseOneWayTls(environment.getAccountId())
+        return CcmV2TlsType.TWO_WAY_TLS == environment.getCcmV2TlsType()
                 ? CcmV2TlsType.TWO_WAY_TLS
                 : CcmV2TlsType.ONE_WAY_TLS;
     }
```

Comment:
I think `environment.getCcmV2TlsType()` is left `null` for all customer environments; see [ccmV2TlsType in environments2.yaml](https://github.infra.cloudera.com/thunderhead/thunderhead/blob/master/services/libs/protocols-yaml/src/main/yaml/environments2.yaml#L2690-L2695) that is only available in the internal CLI build. So I wonder how it will be possible to control this where needed, e.g. GovCloud where `ONE_WAY_TLS` is mandatory AFAIK.